doc/rst/language/evolution.rst

```diff
@@ -13,6 +13,116 @@ useful when migrating programs to the current version of the language.
 The purpose of this flag is to identify  portions of a program that use a
 language feature that has changed meaning.
 
+version 1.21, September 2019
+----------------------------
+
+Version 1.21 made several improvements related to record initialization,
+assignment, and deinitialization.
+
+In summary:
+
+ * Some patterns of default initialization followed by assignment are now
+   converted to initialization. See :ref:`readme-evolution.split-init`.
+ * Some patterns of copy initialization followed by deinitialization are
+   converted to move initialization. See :ref:`readme-evolution.copy-elision`.
+ * The result of a nested call expression can now be deinitialized at the end of
+   the containing statement. See :ref:`readme-evolution.statement-deinit`.
+
+.. _readme-evolution.split-init:
+
+split initialization
+********************
+
+Split initialization a new language feature in 1.21 that is described in
+the language specification - see :ref:`Split_Initialization`.
+
+Consider the following example:
+
+.. code-block:: chapel
+
+  var x: myRecord;    // default-initialization in 1.20
+  x = new myRecord(); // assignment in 1.20 -- initialization in 1.21
+
+In 1.21, instead of default-initializing ``x`` and then assigning to it,
+``x`` will be initialized on the second line.
+
+Note that split initialization also changes the copy and assignment
+behavior of ``out`` intent formal arguments.
+
+Occasionally programs that are written to test assignment (separately
+from copy initialization) need to avoid split initialization. One way to
+do so is to add a mention of the variable immediately after it is
+declared, as in the following code:
+
+.. code-block:: chapel
+
+  var x: myRecord;
+  x; // adding this mention avoid split-initialization
```

Comment:
"avoids" (or "prevents", "squashes", etc.)
It might also be useful to say "instead, the record is default initialized at its declaration point" for clarity?